eZ/Publish/API/Repository/ContentService.php

```diff
@@ -432,4 +432,15 @@ public function newTranslationInfo();
      * @return \eZ\Publish\API\Repository\Values\Content\TranslationValues
      */
     public function newTranslationValues();
+
+    /**
+     * Returns true if the given content type $contentType has content instances.
+     *
+     * @since 6.0.1
+     *
+     * @param \eZ\Publish\API\Repository\Values\ContentType\ContentType $contentType
+     *
+     * @return bool
+     */
+    public function isContentTypeUsed(ContentType $contentType);
```

Comment:
Good arguments from @crevillo. I'd be fine with that.

About using the Content Handler from the ContentType service... unsure. It would make sense that the ContentType handler is able to see by how many items it is used without having to use another handler. Each persistence implementation would be free to reuse whatever blocks it needs.